src/main/java/org/traccar/protocol/FreematicsProtocolDecoder.java

```diff
@@ -146,8 +146,23 @@ private Object decodePosition(
                 case 0x82:
                     position.set(Position.KEY_DEVICE_TEMP, Integer.parseInt(value) * 0.1);
                     break;
+                case 0x104:
+                    position.set(Position.KEY_ENGINE_LOAD, Integer.parseInt(value));
+                    break;
+                case 0x105:
+                    position.set(Position.KEY_COOLANT_TEMP, Integer.parseInt(value));
+                    break;
+                case 0x10c:
+                    position.set(Position.KEY_RPM, Integer.parseInt(value));
+                    break;
+                case 0x10d:
+                    position.set(Position.KEY_OBD_SPEED, UnitsConverter.knotsFromKph(Integer.parseInt(value)));
+                    break;
+                case 0x111:
+                    position.set(Position.KEY_THROTTLE, Integer.parseInt(value));
+                    break;
                 default:
-                    position.set(data[0], value);
+                    position.set(Position.PREFIX_IN + data[0], value);
```

Comment:
I think this should be `Position.PREFIX_IO + key` to be consistent with other decoders. Everything else looks good.

Reply:
Works for me, I've updated the pull request and also added the PID for HDOP.